java/src/main/java/org/rocksdb/LiveFileMetaData.java

```diff
@@ -9,30 +9,40 @@
  * The full set of metadata associated with each SST file.
  */
 public class LiveFileMetaData extends SstFileMetaData {
-  private final byte[] columnFamilyName;
-  private final int level;
 
   /**
    * Called from JNI C++
    */
-  private LiveFileMetaData(
-      final byte[] columnFamilyName,
-      final int level,
-      final String fileName,
-      final String path,
-      final long size,
-      final long smallestSeqno,
-      final long largestSeqno,
-      final byte[] smallestKey,
-      final byte[] largestKey,
-      final long numReadsSampled,
-      final boolean beingCompacted,
-      final long numEntries,
-      final long numDeletions) {
-    super(fileName, path, size, smallestSeqno, largestSeqno, smallestKey,
-        largestKey, numReadsSampled, beingCompacted, numEntries, numDeletions);
-    this.columnFamilyName = columnFamilyName;
-    this.level = level;
+  private LiveFileMetaData(final long nativeHandle) {
+    super(nativeHandle);
+  }
+
+  public LiveFileMetaData() {
+    super(newLiveFileMetaData());
+  }
+
+  /**
+   * @param columnFamilyName the name of the column family
+   * @param level the level at which the file resides
+   * @param fileName the file name
+   * @param path the file path
+   * @param size the size of the file
+   * @param smallestSeqno the smallest sequence number
+   * @param largestSeqno the largest sequence number
+   * @param smallestKey the smallest key
+   * @param largestKey the largest key
+   * @param numReadsSampled the number of reads sampled
+   * @param beingCompacted true if the file is being compacted, false otherwise
+   * @param numEntries the number of entries
+   * @param numDeletions the number of deletions
+   */
+  public LiveFileMetaData(final byte[] columnFamilyName, final int level, final String fileName,
```

Comment:
We need this constructor, so that Java can create an object of **LiveFileMetaData**. Because `ExportImportFilesMetaData` which contains `List<LiveFileMetaData>`, will act as a parameter of  API `CreateColumnFamilyWithImport`.